### PR TITLE
feat(general): Cleanup section code

### PR DESCRIPTION
**Pull request by spotandjake**
This refactors and cleans up the code for the section logic (This will affect our other open prs but I am happy to modify them to fit this new style). We now require styles to be built through the section element but this means we can localize our style logic which keeps things more consistent, additionally I reduced the complexity of the css for our other sections which helps with keeping our code clean and shipping small css bundles.

This code also changes `NewsSection` to support items with an `href` as we will not have the blog to link to when we first release and we might just want to show news elements without more detail.

Additionally this code makes a new `utils.ts` file which we can use to store global utilties such as `formatDate` allowing for consistent use and less code duplication.

### CODE REVIEW

File: layouts/SectionHeader.tsx [hunk 1]
```diff
@@ -1,44 +1,37 @@
...
+      <span></span>
+      <h2>{title}</h2>
+      <span></span>
     </div>
   );
-}
+};
 
-export { SectionLocation, SectionHeaderStyle };
+export default SectionHeader;
```
Comment (Kara-Zor-El) [inline layouts/SectionHeader.tsx:37]: souldn't Alignment be exported?

File: styles/layouts/NewsSection.module.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.container {
+  @apply grid gap-4 p-4 text-white;
+  .newsItem {
+    @apply flex justify-between rounded-xl p-4;
+    @apply border-2 border-gray-500;
+    @apply duration-300 ease-in-out;
+    &:hover {
+      @apply bg-hover-1 border-[#0000];
```
Comment (Kara-Zor-El) [inline styles/layouts/NewsSection.module.scss]: refer to discord discussion on topic

File: styles/layouts/SectionHeader.module.scss [hunk 3]
```diff
@@ -1,83 +1,40 @@
...
-  @apply bg-color-1;
-  & > span {
-    @apply bg-color-3;
-  }
-
-  .frontRoundedCornerContainer,
-  .backRoundedCornerContainer {
-    @apply bg-color-3;
+  // Border Radius
+  span {
```
Comment (Kara-Zor-El) [inline styles/layouts/SectionHeader.module.scss:11]: can we add some more padding to move it over a bit?

File: utils.ts
Comment (Kara-Zor-El) [inline utils.ts:1]: wanna maybe make a utilities folder and add turn this into a formatDate file?

**Review by Kara-Zor-El — APPROVED** [review]
Looks great, a few nits

File: utils.ts
Comment (spotandjake) [reply utils.ts:1]: > wanna maybe make a utilities folder and add turn this into a formatDate file?

I feel like it wouldn't be to hard to make that change later and this means elss module imports. It's not like we have 100 massive utilities or anything. It's mostly going to be little things like `formatDate`, idk `chooseStyle` (maybe), small 10-30 line helpers. And if we run into a case where we find its not large enough we can split it up pretty easily and move to the file way of doing it, I want you to just be able to do `Utils.x` to make it easy to use.

File: styles/layouts/NewsSection.module.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
+.container {
+  @apply p-4 text-white;
+  .newsItem {
+    @apply flex justify-between rounded-xl p-4 m-[0.125rem] mb-5 items-center;
```
Comment (Kara-Zor-El) [inline styles/layouts/NewsSection.module.scss]: `m-0.5` = `m-[0.125rem]`

**Review by Kara-Zor-El — APPROVED** [review]
Make the change and then you're good

**Pull request merged**